docs/source/jit.rst

```diff
@@ -531,13 +531,252 @@ Python-defined Constants
 Debugging
 ~~~~~~~~~
 
-Print things
+Interpreting Graphs
+    TorchScript uses a static single assignment (SSA) intermediate representation
+    (IR) to represent computation. The instructions in this format consist of
+    ATen operators and other primitive operators, including control flow
+    operators for loops and conditionals. As an example::
 
-Use ``USE_PYTHON=0`` to debug in normal python mode
+        @torch.jit.script
+        def foo(len):
+          # type: (int) -> torch.Tensor
+          rv = torch.zeros(3, 4)
+          for i in range(len):
+            if i < 10:
+                rv = rv - 1.0
+            else:
+                rv = rv + 1.0
+          return rv
```

Comment:
Can we come up with a slightly smarter example? 😄This one trivially reduces to 
```python
return rv + (min(0, len - 10) - max(len, 10))
```
It also shadows the `len` builtin, which we definitely shouldn't encourage in the script.